checkov/terraform/runner.py

```diff
@@ -212,9 +244,9 @@ def get_connected_node(self, entity, root_folder) -> Optional[Dict[str, Any]]:
         connected_node_data['resource_address'] = connected_entity_context.get('address')
         return connected_node_data
 
-    def get_graph_checks_report(self, root_folder: str, runner_filter: RunnerFilter) -> Report:
+    def get_graph_checks_report(self, root_folder: str, runner_filter: RunnerFilter, graph: igraph.Graph | None = None) -> Report:
         report = Report(self.check_type)
-        checks_results = self.run_graph_checks_results(runner_filter, self.check_type)
+        checks_results = self.run_graph_checks_results(runner_filter, self.check_type, graph)
```

Comment:
why do you need to send the graph as well?

Reply:
I think it's connected to this comment: https://github.com/bridgecrewio/checkov/pull/5375#discussion_r1273370241
We need to run on a specific graph now